examples/IMDb/IMDb/config.py

```diff
@@ -0,0 +1,14 @@
+import os
+from typing import Dict, Any
+import yaml
+
+def load_od_config() -> Dict[str, Any]:
+    # Load the existing YAML config
+    root = os.path.abspath(os.path.dirname(__file__))
+    file_path = os.path.join(root, 'project_config.yaml')
```

Comment:
I think it might make sense to standardise this. I saw on `armbench` a different naming. Is this name the new default?

Reply:
changing name to "load_config"